package.json

```diff
@@ -16,10 +16,12 @@
   },
   "dependencies": {
     "@daostack/client": "0.1.4",
+    "lodash": "^4.17.11",
```

Comment:
Ramda is already included as a dep, would you mind using that instead of lodash?

Reply:
Opps I didn't realize Ramda was already included.